### PR TITLE
Add Fineco Bank (IT/UK)

### PR DESCRIPTION
**Pull request by rock3r**
Does what it says on the tin. Took the liberty to remove some stray whitespaces from the banking yaml while I was at it

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -556,6 +556,14 @@ websites:
...
       - sms
 
+  - name: Fineco Bank
+    url: https://finecobank.com/
+    img: finecobank.png
+    tfa:
+      - sms
+      - proprietary
+    doc: https://finecobank.com/it/online/sicurezza/
+
```
Comment (Carlgo11) [inline _data/banking.yml:569]:
```suggestion
    regions:
      - it

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/banking.yml (hunk 1 above)
Comment (rock3r) [reply _data/banking.yml:569]: Done — you may want to update the docs tho https://github.com/2factorauth/twofactorauth/wiki/FAQ-New-2FA-Supported-Site

Comment (rock3r) [reply _data/banking.yml:569]: Actually I'll add UK too, since it's both IT and UK

Comment (rock3r) [reply _data/banking.yml:569]: Done that, too https://github.com/2factorauth/twofactorauth/pull/4882/commits/486b31189e59f4ff7c06ae2a6b1b988a327db95f

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]
Not a fan of mixing code cleanup with semantic changes, but here we are.

**Pull request merged**